lib/zenoss/jsonapi/device_router.rb

```diff
@@ -72,6 +72,24 @@ def set_info(opts = {})
         end
       end
 
+      # @param [String] uid required; device id in Zenoss
+      # @param [String] plugins optional; takes a regular expression
```

Comment:
so...does it take regular expression or a string? Also, what does that string or regular expression represent?

Reply:
The takes a string.  The "plugin" is a Modeler Plugin, such as cisco.snmp.Interfaces.